src/main/java/by/dragonsurvivalteam/dragonsurvival/client/handlers/ClientFlightHandler.java

```diff
@@ -51,53 +53,56 @@
 
 import java.util.concurrent.TimeUnit;
 
-/**
- * Used in pair with {@link ServerFlightHandler}
- */
-@Mod.EventBusSubscriber( Dist.CLIENT )
-@SuppressWarnings( "unused" )
-public class ClientFlightHandler{
+/** Used in pair with {@link ServerFlightHandler} */
+@Mod.EventBusSubscriber(Dist.CLIENT)
+@SuppressWarnings("unused")
+public class ClientFlightHandler {
+	@ConfigRange(min = 0, max = 60)
+	@ConfigOption(side = ConfigSide.SERVER, category = "wings", key = "levitationAfterEffect", comment = "For how many seconds wings are disabled after the levitation effect has ended")
```

Comment:
I'm not sure what you're talking about here. Is this adding a mechanic that disables a dragon's flight when getting vanilla levitation effect?

Reply:
In this PR the levitation effect will disable flight / the ability to control it (meaning you will float up like a non-dragon)

After the levitation effect has ended you will still not be able to control your flight - the amount of seconds is handled by this config (0 kinda disables it)
Meaning you will fall down and cant just fly away after levitation has ended

Reply:
that sounds like what we need. Great addition! Have you added a flight effect to slow down as well? Maybe in the future you could add custom arrows/bolas and an effect that forcibly immobilizes the dragon. This was partially implemented with dragon hunters in past versions, but eventually broken and hidden. I think dragons are too safe in the air and that needs to be fixed. In the future